### ISSUE TITLE
Crashed while opening raster layer properties 

### ISSUE THREAD

**Issue by firmanghazali25** [qgis/QGIS#42729]
Hi. Everyone,
Recently, I am using QGIS 3.18.1 and often to get the error message (attached picture) while opening the raster layer properties. How to fix it?. Thank you.

![image](https://user-images.githubusercontent.com/77722849/114125453-b46d5780-9920-11eb-87a5-afd37ed27a80.png)

**Comment by nirvn** [comment]
@firmanghazali25 , this is likely caused by a plugin, try disabling your plugins one by one and open the raster properties window, hopefully you'll identify the problematic plugin.

**Comment by firmanghazali25** [comment]
@nirvn, thank you for your valuable reply. I tried to disable some of the problematic plugins but, it does not work. The QGIS 1.18.1 is still crashed. I decided to re-install the previous version of 1.16.0 and the issue fixed. The raster properties window can be opened properly.

**Comment by gioman** [comment]
>  I tried to disable some of the problematic plugins but, it does not work.

@firmanghazali25 disabling them is sometimes not enough, we had cases of 3rd party plugins that caused QGIS to crash even if disabled. You must try with a new QGIS profile.

**Comment by firmanghazali25** [comment]
@gioman The new QGIS profile means another version of QGIS?

**Comment by gioman** [comment]
> @gioman The new QGIS profile means another version of QGIS?

@firmanghazali25 

settings > user profiles > new profile

**Comment by firmanghazali25** [comment]
Ok. I will try. Thank you.

**Comment by firmanghazali25** [comment]
@gioman, thank you for your suggestion, Its works.

**Comment by gioman** [comment]
> @gioman, thank you for your suggestion, Its works.

@firmanghazali25 you may want to install one by one the plugins in the new profile and find the offending one.

**Comment by firmanghazali25** [comment]
> > @gioman, thank you for your suggestion, Its works.
> 
> @firmanghazali25 you may want to install one by one the plugins in the new profile and find the offending one.

@gioman Ya, I did it. I have to reinstall all the required plugins. But, what is the significant function of using QGIS in default and new profile?

**Comment by gioman** [comment]
> But, what is the significant function of using QGIS in default and new profile?

@firmanghazali25 to start QGIS without anything (old configurations, 3rd party plugins, 3rd party scripts, etc.) that could cause the program to behave in a wrong way.

**Comment by firmanghazali25** [comment]
> > But, what is the significant function of using QGIS in default and new profile?
> 
> @firmanghazali25 to start QGIS without anything (old configurations, 3rd party plugins, 3rd party scripts, etc.) that could cause the program to behave in a wrong way.

@gioman Oh I see.

**Comment by Bananastand-Money** [comment]
I've been getting a similar error to:
https://github.com/qgis/QGIS/issues/41834
https://github.com/qgis/QGIS/issues/41837

I load in the geotiff in my default profile with various plugins but upon selecting "Properties" QGIS would hard crash with no error codes or stack traces. 

I opened a new profile and was able to correctly open properties and began adding my plugins one by one and attempting to re-create the error, however I was unable to re-create the error after adding all of my plugins back. Finally I created a new profile, copied and pasted all the plugins from my default to new profile and everything seems fine.

I've only been able to get it to happen once since starting a new profile and that was after I moved a toolbar, but was then unable to replicate that after installing plugins one by one and moving each individually. I'll keep trying things to see if I can consistently replicate it.

QGIS version
3.18.1-Zürich
QGIS code revision
202f1bf7e5
Compiled against Qt
5.11.2
Running against Qt
5.11.2
Compiled against GDAL/OGR
3.1.4
Running against GDAL/OGR
3.1.4
Compiled against GEOS
3.8.1-CAPI-1.13.3
Running against GEOS
3.8.1-CAPI-1.13.3
Compiled against SQLite
3.29.0
Running against SQLite
3.29.0
PostgreSQL Client Version
11.5
SpatiaLite Version
4.3.0
QWT Version
6.1.3
QScintilla2 Version
2.10.8
Compiled against PROJ
6.3.2
Running against PROJ
Rel. 6.3.2, May 1st, 2020
OS Version
Windows 10 (10.0)
Active python plugins
**db_manager; 
MetaSearch; 
processing; 
slyr_community; 
qProf; 
QGeoloGIS; 
qgis-maptiler-plugin; 
ioGAS; 
geoscience; 
FreehandRasterGeoreferencer; 
enmapboxplugin; 
DataPlotly**
